frappe/custom/doctype/customize_form/test_customize_form.py

```diff
@@ -188,6 +188,19 @@ def test_title_field_pattern(self):
 	def test_core_doctype_customization(self):
 		self.assertRaises(frappe.ValidationError, self.get_customize_form, 'User')
 
+	def test_save_customization_length_field_property(self):
+		# Using Notification Log doctype as it doesn't have any other custom fields
+		d = self.get_customize_form("Notification Log")
+
+		document_name = d.get("fields", {"fieldname": "document_name"})[0]
+		document_name.length = 255
+		d.run_method("save_customization")
+
+		self.assertEqual(frappe.db.get_value("Property Setter",
+			{"doc_type": "Notification Log", "property": "length", "field_name": "document_name"}, "value"), '255')
+
+		self.assertTrue(d.flags.update_db)
```

Comment:
You can check whether `frappe.db.update_db` has been called or the DB column has been updated instead of checking a flag value.